projectile.el

```diff
@@ -502,7 +507,35 @@ just return nil."
         version)
     (error "Cannot determine version without package pkg-info")))
 
-
+;;; Workarounds
+
+(eval-after-load 'etags
+  '(when projectile-etags-tags-completion-table-hack
```

Comment:
Because the way I've done it the user can turn it off, or even forward port the impl in the future. Or we can Toggle it during testing.